modules/apps/headless/headless-builder/headless-builder-web/src/main/resources/META-INF/resources/js/components/EditAPIEndpoint.tsx

```diff
@@ -619,6 +619,7 @@ export default function EditAPIEndpoint({
 										}
 										basePath={basePath}
 										data={localUIData}
+										disabledProperties={['httpMethod']}
```

Comment:
I would change this to something like "isEdition" (we can work on the name if it's not perfect ;)). 
And I wouldn't assign it any value.